src/config.ts

```diff
@@ -175,8 +175,8 @@ export class CacheConfig {
           hasher.update(JSON.stringify(parsed));
 
           parsedKeyFiles.push(cargo_manifest);
-        } catch (_e) {
-          // Fallback to caching them as regular file
+        } catch (e) { // Fallback to caching them as regular file
+          core.warning(`Error parsing Cargo.toml manifest, fallback to caching entire file: ${e}`);
```

Comment:
Instead of using a `warning` which creates an annotation (similar to the issue raised in https://github.com/Swatinem/rust-cache/issues/144), does it make sense to use `debug` rather?

Reply:
I think it's better to issue a warning, so that the warning can be seen even if it's not a debug run.

I don't think it will be as bad as #144 since it's a warning not an error.